chpl/chpl-resources/src/main/resources/environment.properties

```
@@ -16,7 +16,11 @@
 chplUrlBegin=http://localhost:3000
 jndiName=java:comp/env/jdbc/openchpl
 persistenceUnitName=openchpl
-springfox.documentation.swagger.v2.path=/api-docs
+api.version=30.6.0
+api.description=Created by CHPL Development Team. Please submit any questions using the Health IT \
+    Feedback Form and select the "Certified Health IT Products List (CHPL)" category. <br/>\
+    See more at <a href="%s" target="_blank">%s</a>
+api.licenseUrl=https://github.com/chpladmin/chpl-api/blob/staging/LICENSE
```

```suggestion
api.licenseUrl=https://github.com/chpladmin/chpl-api/blob/production/LICENSE
```

I think we should link to the `main` branch, even though it's not our default. Could easily be convinced otherwise. Hmm, maybe link to whatever the latest version location is? And update this line in the same deployment script? I'm trying to think of what is the most "truthy" option, but am not sure which way to go